lib/commander/runner.rb

```diff
@@ -341,8 +341,12 @@ def remove_global_options(options, args)
         past_switch, arg_removed = false, false
         args.delete_if do |arg|
           break if arg == '--'
+
+          arg_equals_index = arg.index('=') if switch_has_arg
+          arg = arg[0, arg_equals_index] if arg_equals_index
```

Comment:
Use just the portion of the argument before the = when comparing against global switches.

Reply:
Might as well put this comment into the code. This method is confusing enough to warrant some extra inline documentation. 😄

Reply:
👍 Added in 9791b4f.